spec/controllers/users_controller_spec.rb

```diff
@@ -48,28 +62,31 @@
       it 'sets flash[:notice]' do
         expect(flash[:notice]).to be_a String
```

Comment:
Rather than expecting on the class type, it should be enough to verify it `to be_present`. That respects duck typing a bit more.